core/src/Cartesian.js

```diff
@@ -1,10 +1,24 @@
 import React, { Fragment } from 'react'
 import { cartesianProduct } from './util'
 
-export default ({ component, ...props }) => {
+export default ({ component, container, ...props }) => {
   const combinations = cartesianProduct(props)
   const Component = component
+  
+  if (container) {
```

Comment:
Should we default container to Fragment so we only have one path for rendering?

Reply:
great thinking. updated.